Cargo.toml

```diff
@@ -26,6 +26,7 @@ members = [
   "bindings/nodejs",
   "bindings/object_store",
   "bindings/python",
+  "bindings/ruby/ext/opendal_ruby",
```

Comment:
🥺🥺🥺, can we place the code in bindings/ruby instead?

Reply:
I will try again. Previously, I always encountered errors related to paths and naming. I also hope to maintain the same style.